app/lib/theme/CustomDiffFields.tsx

```diff
@@ -216,22 +190,18 @@ const showNumberDiff = (
   latestCommittedData: number,
   isMoney: boolean,
   isPercentage: boolean,
-  numberOfDecimalPlaces: number = 0,
-  isAmendmentsAndOtherRevisionsSpecific?: boolean
+  numberOfDecimalPlaces: number = 0
 ): JSX.Element => {
   const decimalScale = isMoney ? 2 : numberOfDecimalPlaces ?? 0;
-  const [diffOldClsName, diffNewClsName] = isAmendmentsAndOtherRevisionsSpecific
-    ? [
-        "diffAmendmentsAndOtherRevisionsOld",
-        "diffAmendmentsAndOtherRevisionsNew",
-      ]
-    : [
-        "diffReviewAndSubmitInformationOld",
-        "diffReviewAndSubmitInformationNew",
-      ];
-  return (
-    <>
-      <span id={id && `${id}-${diffOldClsName}`} className={diffOldClsName}>
+  const [diffOldClsName, diffNewClsName, diffTextClsName] = [
+    "diffOld",
+    "diffNew",
+    "diffText",
+  ];
+  // case 7 ->  111
+  if (oldData && newData && latestCommittedData) {
```

Comment:
In every case that has both old and latestCommitted, we need to check if they're equal before doing anything else because if they are equal, then we're only dealing with two diffs.

This might end up covering case 111, 101, and 100